timezones/zones.py

```diff
@@ -205,8 +206,8 @@ def get_timezones_dict():
     ("+0800", "Asia/Kuala_Lumpur", '(GMT+0800) Kuala Lumpur'),
     ("+0800", "Australia/Perth", '(GMT+0800) Perth'),
     ("+0800", "Singapore", '(GMT+0800) Singapore'),
-    ("+0800", "Asia/Taipei", '(GMT+0800) Taipei'),
     ("+0800", "Asia/Taipei", '(GMT+0800) Ulaan Bataar'),
+    ("+0800", "Asia/Taipei", '(GMT+0800) Taipei'),
```

Comment:
According to the comment from L180-184, when there are more than 1 city of the same timezone name, we should put the city representing this city at the last order, so that it shows up in selection.